### PR TITLE
fix(ins): update images

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Review by RoRoJ — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

File: compute/instances/how-to/create-an-instance.mdx [hunk 1]
```diff
@@ -209,14 +247,14 @@ Select a tab below for instructions on how to create an Instance via either our
...
         | type            | The type of Instance to create                      | Any valid ID of a Scaleway Instance type, e.g. `GP1-S`, `PRO2-M`. Use the [server-type](https://github.com/scaleway/scaleway-cli/blob/master/docs/commands/instance.md#server-type-management-commands) command to get a list of all valid Instance types and their IDs. The default value is `DEV1-S` |
-        | image           | The image to install on the Instance, e.g. a particular OS | Any valid Scaleway image ID or its label, e.g. `544f0add-626b-4e4f-8a96-79fa4414d99a` or`ubutunu_jammy` for the Ubuntu 22.04 Jammy Jellyfish image. The default value is `ubuntu_jammy`. Use the [marketplace](https://github.com/scaleway/scaleway-cli/blob/master/docs/commands/marketplace.md) command to get a list of all available images and their IDs. |
+        | image           | The image to install on the Instance, e.g. a particular OS | Any valid Scaleway image label, e.g. `ubuntu_noble` which is the label for the Ubuntu 24.04 Noble Numbat image. Use the [Marketplace API](https://www.scaleway.com/en/developers/api/marketplace/) endpoint to find available images and their IDs. |
```
Comment (cgilb-scaleway) [inline compute/instances/how-to/create-an-instance.mdx:250]:
```suggestion
        | image           | The image to install on the Instance, e.g. a particular OS | Any valid Scaleway image label, e.g. `ubuntu_noble` which is the label for the Ubuntu 24.04 Noble Numbat image. |
```

File: compute/instances/how-to/create-an-instance.mdx [hunk 2]
```diff
@@ -153,17 +183,17 @@ Select a tab below for instructions on how to create an Instance via either our
...
         | commercial-type | The commercial Instance type to create  (string)    | Any valid ID of a Scaleway commercial Instance type, e.g. `"GP1-S"`, `"PRO2-M"`. Use the [List Instance Types](https://www.scaleway.com/en/developers/api/instance/#path-instance-types-list-instance-types) endpoint to get a list of all valid Instance types and their IDs.   |
-        | image           | The image to install on the Instance, e.g. a particular OS (string) | Any valid Scaleway image ID, e.g. `"544f0add-626b-4e4f-8a96-79fa4414d99a"` which is the ID for the `Ubuntu 22.04 Jammy Jellyfish` image. Use the [Marketplace API](https://www.scaleway.com/en/developers/api/marketplace/) endpoint to find available images and their IDs. |
+        | image           | The image to install on the Instance, e.g. a particular OS (string) | Any valid Scaleway image label, e.g. `ubuntu_noble` which is the label for the Ubuntu 24.04 Noble Numbat image. Use the [Marketplace API](https://www.scaleway.com/en/developers/api/marketplace/) endpoint to find available images and their IDs. |
```
Comment (cgilb-scaleway) [inline compute/instances/how-to/create-an-instance.mdx:186]:
```suggestion
        | image           | The image to install on the Instance, e.g. a particular OS (string) | Any valid Scaleway image label, e.g. `ubuntu_noble` which is the label for the Ubuntu 24.04 Noble Numbat image. |
```

**Comment by cgilb-scaleway** [comment]
There is a duplicate An [SSH key] at the top of the API part

File: compute/instances/how-to/create-an-instance.mdx [hunk 3]
```diff
@@ -128,15 +128,45 @@ Select a tab below for instructions on how to create an Instance via either our
...
 
-    2. Customize the **payload** for your API call. This payload specifies the configuration of the Instance you want to create. We provide an example payload below with all the required parameters, and a table of information to help you adapt it to your needs.
+        ```
+        curl --location "https://api.scaleway.com/marketplace/v2/images" \
+        --header "X-Auth-Token: $SCW_SECRET_KEY"
+
+        {
+          "images": [
+
+            ...
```
[the remark this answers is not in the document]
Comment (ldecarvalho-doc) [reply compute/instances/how-to/create-an-instance.mdx:140]:
```suggestion
            {...}
```
There are no clear guidelines about this anywhere, but I discussed with the team and we decided on going with this.

File: compute/instances/how-to/create-an-instance.mdx [hunk 4]
```diff
@@ -128,15 +128,45 @@ Select a tab below for instructions on how to create an Instance via either our
...
 
-    2. Customize the **payload** for your API call. This payload specifies the configuration of the Instance you want to create. We provide an example payload below with all the required parameters, and a table of information to help you adapt it to your needs.
+        ```
+        curl --location "https://api.scaleway.com/marketplace/v2/images" \
+        --header "X-Auth-Token: $SCW_SECRET_KEY"
+
+        {
+          "images": [
+
+            {...}
```
Comment (ldecarvalho-doc) [inline compute/instances/how-to/create-an-instance.mdx:140]:
```suggestion
            {...},
```

**Pull request merged**